docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -311,8 +310,18 @@ These steps can be performed from within Buildkite's interface. From Buildkite's
 
 <TabItem value='buildkite-semgrep'>
 
+The following configuration creates a CI job that runs scans according to the products you have enabled in Semgrep AppSec Platform. The provided environment variables are commonly needed to correctly configure scans from Buildkite.
 
-The following configuration creates a CI job that runs scans depending on what products you have enabled in Semgrep AppSec Platform.
+This file configures two alternative command paths, one for full scans, and one for diff-aware scans. The latter is used for pull or merge requests.
+
+In order for this configuration to run the correct type of scan, go to the pipeline **Settings** and select the connected source code manager in the left sidebar. In that section, make the following changes:
+
+1. Under **Branch Limiting**, enter your default branch name. You can include any other branch names that require full scans as well, such as `release-*`.
+2. Click **Save Branch Limiting**.
+3. Under the source code manager section, select the box for **Build Pull Requests**. Or for GitLab, when setting up the webhook, ensure that the **Merge request** event is selected.
```

Comment:
```suggestion
3. Under the **source code manager** section, select the box for **Build Pull Requests**. For GitLab, ensure that the **Merge request** event is selected when setting up the webhook.
```

Reply:
The section isn't called "source code manager" so I don't think the first change is right - this is one of the things I wasn't sure how to represent. It's called after whatever the SCM actually is, so "GitHub", or "GitLab", for example.

Reply:
(The fact that you have source code manager settings and then another section under there with settings for the source code manager is...interesting.)

Reply:
Oh, I see. Maybe something like "Under the section labeled with the name of your source code manager...?" If it's super confusing, a screenshot could be helpful.

Reply:
Yeah maybe a screenshot is the way to go. Let me try that out.

Reply:
OK, I ended up reworking this and added screenshots to the first part. The second is totally SCM dependent so I copped out and referred users to Buildkite. 😆 
I'll be out for next week, so please feel free to take this over, clean it up and merge if you're up for it!